examples/lang/bad/scopeerror.mcl

```diff
@@ -0,0 +1,3 @@
+# this file generates a scope error, used in lib/cli_test.go
```

Comment:
all "bad" things should be in test/ dir, not easy to find in examples/

in fact, the *real* tests should have nothing to do with examples/

I have a test case that tests examples just to make sure the examples we submit aren't actually junk, but the real testing should happen in test/